source/messaging/channels-basics.rst

```diff
@@ -16,21 +15,21 @@ Messaging Basics
 You can use either ``_`` or ``*`` for italics and bold text. See the table below for examples.
 
 .. image:: ../images/messagesTable1.png
-   :alt: markdown
+   :alt: Formatting markdown controls the look and feel of text messages.
 
-**Quickly add emojis** by typing ":" followed by two characters, which will open an emoji autocomplete. If the existing emojis don't cover what you want to express, you can also create your own `Custom Emoji <https://docs.mattermost.com/help/settings/custom-emoji.html>`__.
+**Quickly add emojis** by typing ":" followed by two characters, which will open an emoji autocomplete. If the existing emojis don't cover what you want to express, you can also create your own `Custom Emoji <https://docs.mattermost.com/messaging/using-emoji.html#creating-custom-emojis>`__.
 
-**Attach files** by dragging and dropping them into Mattermost or clicking the attachment icon in the text input box.
+**Attach files** by dragging and dropping them into Channels, or by selecting the **Attachment** icon next within the text input box.
```

Comment:
Question: Should this be "into Channels" as the general product, or "into a channel"? It seems like it may be the latter because selecting the Attachment icon needs to be done within a conversation or channel. Also, is it Channels (product) and channels (noun) or always Channels?

Reply:
Great question, @justinegeffen! In this specific instances, you're dragging a file into the Channels product within the Mattermost suite in order to attach the file to a post.

For these terminology updates, I'm applying the following:
- I use "Mattermost Channels" (product noun) for the first instance on a given page, and "Channels" thereafter when referring to the product itself.
- Instances of "Mattermost" (suite noun) remain as-is in cases where we're talking about a Mattermost installation, deployment, system, or instance, and the context isn't product-level specific.
- "channels" (feature noun) are the places where conversations happen.

Reply:
^ All that being said, what are your thoughts for this specific text? I agree that there's ambiguity currently. Perhaps "Attach files by dragging and dropping them into Mattermost Channels, or by selecting the Attachment icon..."

Thoughts?